tests/robot/1_check_airflow.robot

```diff
@@ -5,12 +5,13 @@ Variables           load_variables_from_profiles.py   ${PATH_TO_PROFILES_DIR}
 Library             legion_test.robot.Utils
 
 *** Test Cases ***
-Connect to Airflow and check it
-    [Documentation]  Connect to Airflow and check status
+Check test dags should not fail
+    [Documentation]  Connect to Airflow and check the status of test dags are not failed
     [Tags]  airflow  airflow-api
     :FOR    ${enclave}    IN    @{ENCLAVES}
     \  Connect to enclave Airflow     ${enclave}
         ${dags} =                   Find Airflow DAGs
         Should not be empty         ${dags}
         ${failed_dags} =            Get failed Airflow DAGs
-        should be empty             ${failed_dags}
+        Should Not Contain          ${failed_dags}    example_python_work
+        Should Not Contain          ${failed_dags}    s3_connection_test
```

Comment:
Another approach was to use words "test" and "example", but I think this implementation with the exact tests dag ids is better and more stable.